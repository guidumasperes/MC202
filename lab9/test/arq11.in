888 28
1 R-917 1234
1 N-190 982
1 C-572 1742
1 G-644 125
3 C-572
2 Q-332
1 N-320 339
4 1179
3 N-320
1 S-947 462
1 L-326 1459
1 P-523 730
1 E-831 1039
3 N-190
9
