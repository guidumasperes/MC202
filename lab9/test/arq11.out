chave removida!
chave nao encontrada!
wubba lubba dub dub!
chave removida!
chave removida!
